src/ManageCourses.ApiClient/GitVersion.txt

```diff
@@ -1 +1 @@
-0.5.0
\ No newline at end of file
+0.5.1
```

Comment:
I think this should be 0.6.0 per http://semver.org

minor

Reply:
I do see what you did there :+1: